lib/plugins/inspec-habitat/lib/inspec-habitat/profile.rb

```diff
@@ -1,260 +1,243 @@
 # encoding: utf-8
-# author: Adam Leff
 
 require 'inspec/profile_vendor'
 require 'mixlib/shellout'
 require 'tomlrb'
+require 'ostruct'
 
 module InspecPlugins
   module Habitat
     class Profile
-      attr_reader :options, :path, :profile
-
-      def self.create(path, options = {})
-        creator = new(path, options)
-        hart_file = creator.create
-        creator.copy(hart_file)
-      ensure
-        creator.delete_work_dir
-      end
-
-      def self.setup(path)
-        new(path).setup
-      end
-
-      def self.upload(path, options = {})
-        uploader = new(path, options)
-        uploader.upload
-      ensure
-        uploader.delete_work_dir
-      end
-
+      attr_reader :logger
       def initialize(path, options = {})
         @path    = path
         @options = options
-        @cli_config = nil
-
-        log_level = options.fetch('log_level', 'info')
-        @log = Inspec::Log
-        @log.level(log_level.to_sym)
+        @logger  = Inspec::Log
```

Comment:
inspec-init uses Inspec::UI to provide user feedback.  I think it would be better if inspec-habitat did the same.  That said, we could add that as a subtask on #3715 and take this as-is; it just feels bad to re-write a plugin and not use the UI system.

Reply:
I agree! Let's get this in as-is given the time constraint of me changing teams. I see you've already updated the issue on #3715, but let me know if there is anything else I can do.